commitlint/helpers.ts

```diff
@@ -19,16 +19,10 @@ export abstract class Helpers {
     public static convertAnyToString(
         potentialString: any,
         paramName: string
-    ): string {
+    ): any {
```

Comment:
@realmarv look at the name of this function, the point of it is to not deal with an `any` object! null should be a valid string, and if it's not, then you can use `string?` I think, not sure if it's `?` or other char

Reply:
Done